src/index.js

```diff
@@ -1,30 +1,37 @@
 import './style.css';
 import UI from './modules/ui.js';
+import APIScores from './modules/API.js';
 
-const { v4: uuidv4 } = require('uuid');
+const apiUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api';
+const gameId = 'ewg5IotOpaSFHt0HGmPt';
+const scoresList = document.querySelector('.scores-list');
+const scoreInput = document.querySelector('#score-input');
+const nameInput = document.querySelector('#name-input');
+const refreshButton = document.querySelector('#refresh-button');
+const form = document.querySelector('.form');
 
-const items = [
-  {
-    name: 'Giuseppe',
-    score: '100',
-    id: uuidv4(),
-  },
+// FUNCTIONS
 
-  {
-    name: 'Monacho',
-    score: '12000000000000',
-    id: uuidv4(),
-  },
+async function refreshScores() {
+  const data = await APIScores.getScores(apiUrl, gameId);
+  scoresList.innerHTML = '';
+  data.forEach((item) => UI.renderScoreList(item));
+}
```

Comment:
According to the project requirements you need to use arrow functions exclusively.

<img width="476" alt="Screen Shot 2022-06-15 at 21 14 27" src="https://user-images.githubusercontent.com/82426682/173976610-6a5f07d8-977e-4b6d-aab3-5f752f8ea321.png">

Change this function to an arrow function, please